lib/token.h

```diff
@@ -896,7 +896,7 @@ class CPPCHECKLIB Token {
      * @param attributes Print attributes of tokens like "unsigned" in front of it.
      * @param macro Prints $ in front of the token if it was expanded from a macro.
      */
-    void stringify(std::ostream& os, bool varid, bool attributes, bool macro) const;
+    void stringify(std::string& os, bool varid, bool attributes, bool macro) const;
```

Comment:
Yes since there's a case where `std::cout` is passed into it.

I will revisit it and approach it from another angle with knowledge about the hash usage. Also there's `std::stringstream` usage in the mathlib as well which would actually affect the analysis.

Reply:
> Yes since there's a case where std::cout is passed into it.

I do not see the problem. Instead of:

    scope->enumType->stringify(std::cout, false, true, false);

you would write:

    std::cout << scope->enumType->stringify(false, true, false);

Reply:
Will do when I take another look at the `--bug-hunting` performance. Wanted to flush out the current changes first.